padertorch/data/fragmenter.py

```diff
@@ -107,3 +107,95 @@ def fragment(key, x):
             fragment = deflatten(fragment)
             fragments.append(fragment)
         return fragments
+
+
+def get_segment_offsets(num_samples, segment_length, step=None,
```

Comment:
How about to sync the names with the stft?

Reply:
You mean window_length and shift? Sure, although I might go with length and shift.